docs/guides/integrate-external-app.md

```
@@ -0,0 +1,240 @@
+- **Two goals**: board health (max 3 blocked) and auto-mode active
+- **Three actions**: alert on blocked, dispatch Ava to triage, alert auto-mode off
+
+Files:
+- `workspace/domains.yaml` — domain definitions
+- `workspace/agents.yaml` — A2A agent registration
+- `workspace/goals.yaml` — Ava goals section
+- `workspace/actions.yaml` — Ava actions section
+
+## Dispatch flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)